package.json

```diff
@@ -160,6 +162,7 @@
         "expect": "^23.6.0",
         "fake-indexeddb": "^2.0.4",
         "firebase-tools": "^7.3.2",
+        "firebase-backend": "git+https://github.com/worldbrain/firebase-backend.git#feature/auth",
```

Comment:
Do we need to include this whole package? Does it bloat the size of the extension in any way?

Reply:
I removed this package indeed and replicated one type across the packages instead. To do is to split it out if necessary, but would be good to discuss that first.